File: frontend/lib/types/voice.ts
export interface Voice {
  id: string;
  name: string;
  minimax_voice_id: string;
  language: string;
  tags: string[];
  is_preset: boolean;
  sample_local_path: string | null;
  created_at: string;
}

export interface VoiceListResponse {
  items: Voice[];
  total: number;
}

File: src/api/schemas/voice.py
"""Schemas Pydantic de la biblioteca de voces."""

from __future__ import annotations

from pydantic import BaseModel, Field


class VoiceResponse(BaseModel):
    id: str
    name: str
    minimax_voice_id: str
    language: str
    tags: list[str] = Field(default_factory=list)
    is_preset: bool = False
    sample_local_path: str | None = None
    created_at: str


class VoiceListResponse(BaseModel):
    items: list[VoiceResponse]
    total: int
